modules/monitoring/manifests/checks/accounts_slos.pp

```diff
@@ -0,0 +1,32 @@
+# == Class: monitoring::checks::accounts_slos
+#
+# Nagios alerts for Account Team SLOs
+#
+# === Parameters
+#
+# [*enabled*]
+#   This variable enables to define whether to perform Account SLO checks. [This is set to 'true' by default.]
+class monitoring::checks::accounts_slos (
+  $enabled = true,
+) {
+  if $enabled {
+    # Warn when Accounts has less than 50% of our error budget remaining
+    $error_budget_warning = 50
+    # Declare critical when Accounts has less than 25% of our error budget remaining
+    $error_budget_critical = 25
+
+    $latency_bad_metric = 'transformNull(offset(scale(removeBelowValue(maxSeries(stats.timers.*.nginx_logs.account-api.time_request.upper_90),0.850),0),1))'
+    $latency_all_metric = 'offset(scale(transformNull(maxSeries(stats.timers.*.nginx_logs.account-api.time_request.upper_90)),0),1)'
+    $latency_error_budget = '0.01'
+
+    @@icinga::check::graphite { 'check_error_budget_remaining':
+      target         => "asPercent(scale(offset(divideSeries(integral(${latency_bad_metric}),integral(${latency_all_metric})),-${latency_error_budget}),-1),${latency_error_budget})",
+      warning        => $error_budget_warning,
+      critical       => $error_budget_critical,
```

Comment:
I think the behaviour is "alert if target is over threshold", but we want the opposite.

So instead of having a value from 0 to 100, let's have a value from -100 to 0:

```
target   => "scale(..., -1)",
warning  => "-${error_budget_warning}",
critical => "-${error_budget_critical}",
```